services/media/service/src/domains/common/handlers/delete-entity-command-handler.ts

```diff
@@ -1,67 +1,68 @@
-import { LoginPgPool, transactionWithContext } from '@axinom/mosaic-db-common';
-import { Broker, MessageInfo } from '@axinom/mosaic-message-bus';
-import { assertDictionary } from '@axinom/mosaic-service-common';
+import { assertDictionary, Logger } from '@axinom/mosaic-service-common';
+import {
+  StoreOutboxMessage,
+  TypedTransactionalMessage,
+} from '@axinom/mosaic-transactional-inbox-outbox';
 import {
   DeleteEntityCommand,
   EntityDeletedEvent,
   MediaServiceMessagingSettings,
 } from 'media-messages';
-import { deletes, IsolationLevel } from 'zapatos/db';
+import { ClientBase } from 'pg';
+import { deletes } from 'zapatos/db';
 import { Table } from 'zapatos/schema';
 import { Config } from '../../../common';
-import { MediaGuardedMessageHandler } from '../../../messaging';
+import { MediaGuardedTransactionalInboxMessageHandler } from '../../../messaging';
 
-export class DeleteEntityCommandHandler extends MediaGuardedMessageHandler<DeleteEntityCommand> {
```

Comment:
Would also change the file name of this file and related test file to not includ `command`